src/client/conn/http2.rs

```diff
@@ -20,6 +20,7 @@ use crate::proto;
 use crate::rt::{Executor, Timer};
 
 /// The sender side of an established connection.
+#[derive(Clone)]
```

Comment:
I think you need a custom impl of clone here since this will generate a `impl<B: Clone> Clone for SendRequest<B>` which we don't care if B is clone at all. I believe the UnboundedSender type has the same type of bound that doesn't have clone dependent on the T: clone.

Reply:
Indeed.

Forgot that there is a generic parameter (and probably would have assumed it would add `where dispatch::UnboundedSender<Request<B>, Response<IncomingBody>> : Clone`, not just `where B : Clone` anyway).

Re-uploading (also rebased).